Configuration/PyReleaseValidation/python/relval_production.py

```diff
@@ -31,9 +31,11 @@
 
 workflows[1041] = ['',['RunExpressPhy2017F','TIER0EXPPPSCAL','ALCASPLITPPSCAL','ALCAHARVDPPSCAL']]
 workflows[1042] = ['',['RunExpressPhy2017F','TIER0EXPPPSCALALIG','ALCASPLITPPSALIG','ALCAHARVDPPSALIG']]
-workflows[1043] = ['',['RunRawPPS2022B','TIER0EXPPPSCALRUN3','ALCASPLITPPSCALSAMPIC','ALCAHARVDPPSCALSAMPIC']]
-workflows[1044] = ['',['RunRawPPS2022B','TIER0EXPPPSCALRUN3','ALCASPLITPPSCALRUN3','ALCAHARVDPPSCALRUN3']]
-workflows[1045] = ['',['RunRawPPS2022B','TIER0EXPPPSCALRUN3','ALCASPLITPPSALIGRUN3','ALCAHARVDPPSALIGRUN3']]
+workflows[1046] = ['',['RunRawPPS2024E','TIER0EXPPPSCALRUN3','ALCASPLITPPSCALSAMPIC','ALCAHARVDPPSCALSAMPIC']]
+workflows[1047] = ['',['RunRawPPS2024E','TIER0EXPPPSCALRUN3','ALCASPLITPPSCALRUN3','ALCAHARVDPPSCALRUN3']]
+workflows[1048] = ['',['RunRawPPS2024E','TIER0EXPPPSCALRUN3','ALCASPLITPPSALIGRUN3','ALCAHARVDPPSALIGRUN3']]
```

Comment:
```suggestion
workflows[1046] = ['',['RunRawPPS2024D','TIER0EXPPPSCALRUN3','ALCASPLITPPSCALSAMPIC','ALCAHARVDPPSCALSAMPIC']]
workflows[1047] = ['',['RunRawPPS2024D','TIER0EXPPPSCALRUN3','ALCASPLITPPSCALRUN3','ALCAHARVDPPSCALRUN3']]
workflows[1048] = ['',['RunRawPPS2024D','TIER0EXPPPSCALRUN3','ALCASPLITPPSALIGRUN3','ALCAHARVDPPSALIGRUN3']]
```

Reply:
Already done in https://github.com/cms-sw/cmssw/pull/45456/commits/f8064642d22ea7ec408355106ea4ec246d13e32b